### ISSUE TITLE
Data Update

### ISSUE THREAD

**Issue by ghost** [rd11490/owl_tutorials#1]
Someone made a new export of the .csv data: https://github.com/GoldenAppleGuy/stats_lab_api/blob/main/phs-2023-GAGWeek19.zip
It has some unexplainable errors such as prot3ct showing up on OWLCharts but not on this .csv, but it should be better overall since it covers more of the Asia data.

Either way, you can run the API again if you want.

**Comment by ghost** [comment]
Obviously, it will lose data like first death and first kills, so that will have to somehow be re-calculated all again

**Comment by Askardhiksa** [comment]
Also, there are some blatant bugs like this: https://media.discordapp.net/attachments/1133933057091571722/1180982994857644133/image.png?ex=657f671f&is=656cf21f&hm=246ebe624f0c288dcf36c2464193e8a88fe7a1211f55011edf84fca46b959d20&=&format=webp&quality=lossless&width=1585&height=888

**Comment by ghost** [comment]
Also, there are some blatant bugs like this: https://media.discordapp.net/attachments/1133933057091571722/1180982994857644133/image.png?ex=657f671f&is=656cf21f&hm=246ebe624f0c288dcf36c2464193e8a88fe7a1211f55011edf84fca46b959d20&=&format=webp&quality=lossless&width=1585&height=888

It appears to be fixed on the new .csv linked above
